### PR TITLE
Rename access consents, data consents, and access consent registry

### PR DESCRIPTION
**Pull request by justinwb**
Per agreement by the panel, this PR renames:

- Access Consents -> Access Authorizations
- Data Consents -> Data Authorizations
- Access Consent Registry -> Access Registry

Ancillary references, links, properties, shapes, shape trees, classes, and snippets have also been updated.

### CODE REVIEW

File: proposals/specification/interop.shex [hunk 1]
```diff
@@ -16,7 +16,7 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 <#RegistrySetShape> {
   a [ interop:RegistrySet ] ;
   interop:hasAgentRegistry                  IRI  // shex:reference <#AgentRegistryShape> ;
-  interop:hasAccessConsentRegistry          IRI  // shex:reference <#AccessConsentRegistryShape> ;
+  interop:hasAccessRegistry          IRI  // shex:reference <#AccessRegistryShape> ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.shex:19]:
```suggestion
  interop:hasAccessAuthorizationRegistry          IRI  // shex:reference <#AccessAuthorizationRegistryShape> ;
```

Only if we also change other places

File: proposals/specification/interop.shex [hunk 2]
```diff
@@ -118,72 +118,72 @@ PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
   interop:hasAccessNeedGroup                @<#AccessNeedGroupShape>+
 }
 
-<#AccessConsentRegistryShape> {
-  a [ interop:AccessConsentRegistry ] ;
-  interop:hasAccessConsent                   IRI* // shex:reference <#AccessConsentShape>
+<#AccessRegistryShape> {
+  a [ interop:AccessRegistry ] ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.shex:122]:
```suggestion
<#AccessAuthorizationRegistryShape> {
  a [ interop:AccessAuthorizationRegistry ] ;
```
Only if we also change other places

File: proposals/specification/interop.tree [hunk 3]
```diff
@@ -88,22 +88,22 @@ PREFIX interops: <http://www.w3.org/ns/solid/interop-schema#>
   st:expectsType    st:Container ;
   st:shape          interops:DataRegistrationShape .
 
-# Access Consent Registry
+# Access Registry
 ##########################################################
 
-<#AccessConsentRegistryTree>
+<#AccessRegistryTree>
```
Comment (elf-pavlik) [inline proposals/specification/interop.tree:94]:
```suggestion
<#AccessAuthorizationRegistryTree>
```
Only if we also change other places

File: proposals/specification/interop.tree [hunk 4]
```diff
@@ -88,22 +88,22 @@ PREFIX interops: <http://www.w3.org/ns/solid/interop-schema#>
...
   a st:ShapeTree ;
   st:expectsType    st:Container ;
-  st:shape          interops:AccessConsentRegistryShape ;
-  st:contains       <#AccessConsentTree> ,
-                    <#DataConsentTree> .
+  st:shape          interops:AccessRegistryShape ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.tree:97]:
```suggestion
  st:shape          interops:AccessAuthorizationRegistryShape ;
```

Only if we also change other places

File: proposals/specification/interop.ttl [hunk 5]
```diff
@@ -81,13 +81,13 @@ rdfs:label "Terms for application interoperability in Solid"@en .
   rdfs:range :AgentRegistry ;
   rdfs:label "Has Agent Registry"@en .
 
-:hasAccessConsentRegistry
+:hasAccessRegistry
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:84]:
```suggestion
:hasAccessAuthorizationRegistry
```
Only if we also change other places

File: proposals/specification/interop.ttl [hunk 6]
```diff
@@ -81,13 +81,13 @@ rdfs:label "Terms for application interoperability in Solid"@en .
...
   rdfs:subPropertyOf :hasRegistry ;
-  rdfs:comment "Links to an Access Consent Registry in a registry set"@en ;
+  rdfs:comment "Links to an Access Authorization Registry in a registry set"@en ;
   rdfs:isDefinedBy <> ;
   rdfs:domain :RegistrySet ;
-  rdfs:range :AccessConsentRegistry ;
-  rdfs:label "Has Access Consent Registry"@en .
+  rdfs:range :AccessRegistry ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:89]:
```suggestion
  rdfs:range :AccessAuthorizationRegistry ;
```
Only if we also change other places

File: proposals/specification/interop.ttl [hunk 7]
```diff
@@ -234,99 +234,99 @@ rdfs:label "Terms for application interoperability in Solid"@en .
...
-:hasAccessConsent
+:hasAccessAuthorization
   rdfs:subPropertyOf :hasRegistration ;
-  rdfs:comment "Links to an AccessConsent in an Access Consent Registry"@en ;
+  rdfs:comment "Links to an AccessAuthorization in an Access Authorization Registry"@en ;
   rdfs:isDefinedBy <> ;
-  rdfs:domain [ owl:unionOf ( :AccessConsentRegistry ) ] ;
-  rdfs:range :AccessConsent ;
-  rdfs:label "Has Access Consent"@en .
+  rdfs:domain [ owl:unionOf ( :AccessRegistry ) ] ;
```
Comment (elf-pavlik) [inline proposals/specification/interop.ttl:253]:
```suggestion
  rdfs:domain :AccessAuthorizationRegistry ;
```
Only if we also change other places

Comment (elf-pavlik) [inline proposals/specification/interop.ttl:253]:
```suggestion
  rdfs:domain :AccessRegistry ;
```

Even without renaming we probably don't need a union of one.

File: proposals/specification/snippets/alice.example/access.ttl [hunk 8]
```diff
@@ -0,0 +1,11 @@
+PREFIX interop: <http://www.w3.org/ns/solid/interop#>
+PREFIX alice-access: <https://alice.example/access/>
+
+alice-access:
+  a interop:AccessRegistry;
```
Comment (elf-pavlik) [inline proposals/specification/snippets/alice.example/access.ttl:5]:
```suggestion
  a interop:AccessAuthorizationRegistry;
```

Only if we also change other places

File: proposals/specification/snippets/alice.example/registries.ttl [hunk 9]
```diff
@@ -6,7 +6,7 @@ PREFIX alice-personal: <https://personal.alice.example/>
 alice:registries
   a interop:RegistrySet;
   interop:hasAgentRegistry alice:agents\/ ;
-  interop:hasAccessConsentRegistry alice:consents\/ ;
+  interop:hasAccessRegistry alice:access\/ ;
```
Comment (elf-pavlik) [inline proposals/specification/snippets/alice.example/registries.ttl:9]:
```suggestion
  interop:hasAccessAuthorizationRegistry alice:access\/ ;
```
Only if we also change other places

File: proposals/specification/social-agent.bs [hunk 10]
```diff
@@ -111,9 +111,9 @@ consented [=Applications=].
       <td>[=Agent Registry=] for the [=Social Agent=]</td>
     </tr>
     <tr>
-      <td>hasAccessConsentRegistry</td>
-      <td>[AccessConsentRegistry](#classAccessConsentRegistry)</td>
-      <td>[=Access Consent Registry=] for the [=Social Agent=]</td>
+      <td>hasAccessRegistry</td>
+      <td>[AccessRegistry](#classAccessRegistry)</td>
```
Comment (elf-pavlik) [inline proposals/specification/social-agent.bs:115]:
```suggestion
      <td>hasAccessAuthorizationRegistry</td>
      <td>[AccessAuthorizationRegistry](#classAccessAuthorizationRegistry)</td>
```
Only if we also change other places

File: proposals/specification/interop.jsonld [hunk 11]
```diff
@@ -142,12 +142,12 @@
       "@id": "interop:delegationOfGrant",
       "@type": "@id"
     },
-    "hasAccessConsentRegistry": {
-      "@id": "interop:hasAccessConsentRegistry",
+    "hasAccessRegistry": {
+      "@id": "interop:hasAccessRegistry",
```
Comment (elf-pavlik) [inline proposals/specification/interop.jsonld:146]:
```suggestion
    "hasAccessAuthorizationRegistry": {
      "@id": "interop:hasAccessAuthorizationRegistry",
```
Only if we also change other places

**Review by elf-pavlik — APPROVED** [review]
LGTM today we could decide `AccessRegistry` vs `AccessAuthorizationRegistry`

**Comment by justinwb** [comment]
@elf-pavlik agree with you point on `Access Registry` possibly leading to confusion on why Access Grants aren't stored there. AccessAuthorizationRegistry is a bit of a mouthful - thoughts on `AuthorizationRegistry` instead?

**Comment by elf-pavlik** [comment]
👍  `AuthorizationRegistry`

**Pull request merged**